src/kili/mutations/asset/__init__.py

```diff
@@ -58,16 +58,16 @@ def append_many_to_dataset(
                 - For a `TEXT` project, the content can be either raw text, or URLs to TEXT assets.
                 - For an `IMAGE` / `PDF` project, the content can be either URLs or paths to existing
                     images/pdf on your computer.
```

Comment:
- For a `VIDEO` project, the content can be either URLs pointing to videos hosted on a web server or paths to existing video files on your computer. If you want to import video from frames, look at the json_content section below